src/main/model/CurlCommand.java

```diff
@@ -0,0 +1,107 @@
+package main.model;
+/* Model for the curl command: \"curl -X 'GET' \
+       'https://financial-integration.service.cnqr.tech/fi/v1/companies/1/transactiontypes/expense/transactions?limit=0&docId=%s&ignoreDocumentStatus=yes' \
+       -H 'accept: application/json' \
+       -H 'documentFormatAs: json' \
+      -H 'Authorization: Bearer $JWT'\" \
+       > ~/Downloads/%s.json;
+       Additional note:
+When you run a Python script with the -H flag, it requests that the security policy sets the HOME environment
+variable to the home directory of the target user (usually root by default) as specified in the password database.
+This behavior ensures that the script runs with the correct user context, including the appropriate home directory.
+*/
+public class CurlCommand {
+    private String method;
+    //method describes what the curl command will do "get"
```

Comment:
I was thinking more of an update to the variable name itself to make it descriptive - same with the others, always making them as descriptive as possible (and reasonable heh).